tools/ci_build/github/azure-pipelines/templates/download-deps.yml

```diff
@@ -11,7 +11,7 @@ steps:
       packageType: upack
       feed: '/7424c8e4-5c62-490e-95c4-79446f31017c'
       definition: '517c4f6f-5437-4392-a70d-4f15ec5be2f0'
-      version: 1.0.145
+      version: 1.0.149
```

Comment:
>149 [](http://example.com/codeflow?start=19&length=3)

generated by a tool?

Reply:
This is set according to instructions at the top of deps.txt which includes uploading artifacts with the latest version number in the feed (148) + 1.